casepro/contacts/models.py

```diff
@@ -184,6 +184,17 @@ def get_or_create(cls, org, uuid, name=None):
             return contact
 
     @classmethod
+    def get_or_create_from_urn(cls, org, path, urn, name=None):
```

Comment:
Our current nomenclature is urn=scheme+path, but I think it's easiest if we pass around URNs as the complete string, so this becomes just `get_or_create_from_urn(cls, org, urn, name=None)` where `urn` is a string like `tel:+123456`